R-packages/evalcast/R/plot_trajectory.R

```diff
@@ -28,7 +28,8 @@
 plot_trajectory <- function(list_of_predictions_cards,
                             first_day = "2020-07-01",
                             last_day = Sys.Date(),
-                            alpha = .2)
+                            alpha = .2,
+                           page_facet = 1)
```

Comment:
Let's also pass some extra arguments here `ncol=4` and `nrow=6` or some reasonable defaults to be used below.